recipes/audiofile/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.1.1":
+    url: "https://github.com/adamstark/AudioFile/archive/1.1.1.tar.gz"
```

Comment:
It seems this new version could be using some more modern C++ features or standard and it is failing to compile with some gcc versions. I have tested it with Windows, and I see some warnings that some code is C++17.

It might be necessary to introduce a minimum compiler/cppstd for 1.1.1. Do you think you could do it? If not please ask for help.